hr_public_holidays/__manifest__.py

```diff
@@ -16,6 +16,8 @@
     ],
     'data': [
         'security/ir.model.access.csv',
+        'data/ir_config_parameter.xml',
+        'static/src/xml/assets.xml',
```

Comment:
sorry, missed this path: should not be in static. `templates/assets.xml` works better :)

Reply:
yup, seems neat